README.md

```diff
@@ -26,41 +26,32 @@ yarn test
 ```
 To run individual test files, use
 ```
-npx hardhat test <PATH_TO_TEST>
+yarn hardhat test <PATH_TO_TEST>
 ```
 ## Deployment
 Hardhat does not support native deployments. To aid in deployment, hardhat-deploy is being used. You can see the deployment scripts in the `/deploy` directory.
 
-To run deploys
+Tracer also supports programatic verification of contracts during deployment. Use the `hardhat.config.js` file to set your Etherscan API key in order for this to be enabled.
+
+To run deploys to a live network
+```
+yarn hardhat deploy --network NETWORK --tags LiveDeploy
 ```
-npx hardhat deploy
+This will run the LiveDeployment script which is made to deploy to a real network. If there is any issues with verification, you may run
```

Comment:
```suggestion
This will run the `LiveDeployment` script which is made to deploy to a real network. If there are any issues with verification, you may run
```